javascript/chronometer.js

```diff
@@ -1,26 +1,65 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
+
   startClick(callback) {
-    // ... your code goes here
+   this.currentTime = 0;
+   let intervalId = setInterval(() =>{
+     this.currentTime += 1;
+   }, 1000);  
+
+   return this.currentTime;
```

Comment:
There is not need to return the currentTime, but this doesn't break anything. It is just not needed.